QuizApp/AppDelegate.swift

```diff
@@ -5,14 +5,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
Fix to follow [styleguide](https://github.com/fiveagency/ios-swift-guide/tree/master/Code%20Style#function-declarations).